charts/stable/shinobi/values.yaml

```diff
@@ -0,0 +1,62 @@
+#
+# IMPORTANT NOTE
+#
+# This chart inherits from our common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+#
+
+image:
+  # -- image repository
+  repository: shinobisystems/shinobi
+  # -- image tag
+  # @default -- chart.appVersion
+  tag: dev
+  # -- image pull policy
+  pullPolicy: Always
+
+# -- environment variables. See more environment variables in the [shinobi documentation](https://shinobi.video/docs/).
+# @default -- See below
+env:
+  # -- Set the container timezone
+  TZ: UTC
+  # PLUGIN_KEYS:
+  # SSL_ENABLED:
```

Comment:
If you specify the env vars here, please also explain what they do. Otherwise we can just leave them out (but keep the reference to the upstream docs of course).